src/Stripe/Services/Invoices/StripeInvoiceService.cs

```diff
@@ -53,6 +53,14 @@ public virtual IEnumerable<StripeInvoice> List(StripeInvoiceListOptions listOpti
             );
         }
 
+        public virtual IEnumerable<StripeInvoiceLineItem> ListLineItems(string invoiceId, StripeInvoiceListLineItemsOptions listOptions = null, StripeRequestOptions requestOptions = null)
```

Comment:
this should just be called List, to match the rest of the library. And these two methods should be moved to a new InvoiceLineItemService - this follows their documentation exactly. If it doesn't - trust me - I get many emails. ;)